tests/test_redshift.py

```diff
@@ -182,6 +182,20 @@ def test_redshift_snapshot(self):
         resources = p.run()
         self.assertEqual(len(resources), 1)
 
+    def test_redshift_enhanced_vpc_routing(self):
+        factory = self.replay_flight_data('test_redshift_enhanced_vpc_routing')
+        p = self.load_policy({
+            'name': 'redshift-enhanced-vpc-routing',
+            'resource': 'redshift',
+            'filters': [
+                {'type': 'value',
+                 'key': 'EnhancedVpcRouting',
+                 'value': True}],
+            'actions': [{'type': 'enhanced-vpc-routing', 'enabled': False}]},
+            session_factory=factory)
+        resources = p.run()
+        self.assertEqual(len(resources), 1)
```

Comment:
It seems the Redshift cluster goes into a **modifying** status for a while before the attribute change sticks. I could fake a `describe_clusters` response in the unit test but that's not gaining us anything 😄.